.mergify.yml

```diff
@@ -20,3 +20,10 @@ pull_request_rules:
         method: squash
         strict: smart+fasttrack
         commit_message: title+body
+  - name: automatic approval for Dependabot pull requests
+    conditions:
+      - author~=^dependabot(|-preview)\[bot\]$
+    actions:
+      review:
+        type: APPROVE
+        message: Automatically approving dependabot
```

Comment:
I think we should combine both rules because we want to first review and  then squash merge. 
So something like this probably:
```yml
  - name: Automatic merge for Dependabot pull requests
    conditions:
      - author~=^dependabot(|-preview)\[bot\]$
      - status-success~=^test \([0-9]+\.x, ubuntu-latest\)$
      - base=master
    actions:
      - review:
        type: APPROVE
        message: Automatically approving dependabot
      - merge:
        method: squash
        strict: smart+fasttrack
        commit_message: title+body
```